sandbox/src/train.ts

```diff
@@ -135,7 +135,25 @@ function computeParameters(
 		// Do not read from memory after `Stop` is posted!
 		pointer: progress.pointer(),
 	} satisfies ProgressMessage)
-	let parameters = fsrs.computeParametersAnki(cids, eases, ids, types, progress)
+	// MDN states:
+	//     The number of minutes returned by getTimezoneOffset() is positive if the
+	//     local time zone is behind UTC, and negative if the local time zone is ahead of UTC.
+	// This is useful when going from our timezone to UTC; our local time + offset = UTC.
+	// However, we want to go from UTC to our timezone. (Anki revlog id is UTC, and we want to convert it to local time.)
+	// So we flip the sign of `getTimezoneOffset()`.
+	let timeZoneMinutesOffset = new Date().getTimezoneOffset() * -1
+	let ankiNextDayStartsAtMinutes = 4 * 60
+	let parameters = fsrs.computeParametersAnki(
+		// We subtract ankiNextDayStartsAtMinutes because
+		// we want a review done at, say, 1am to be done for the *prior* day.
+		// Adding would move it into the future.
+		timeZoneMinutesOffset - ankiNextDayStartsAtMinutes,
```

Comment:
Yeah, it's intended. Actually, we hope the user doesn't do their reviews around the `NextDayStarts`, when they should be sleeping.

Reply:
> If my timezone is UTC with a `utc_offset = 0` and `next_day_starts_at = 4`, then the `minute_offset = 240 minutes`.

To be exceedingly pedantic, `minute_offset = -240 minutes`; i.e. it's negative. This can be seen in [the original code](https://github.com/open-spaced-repetition/fsrs-rs/blame/64b5f518cff07051228461d4275f9501382a4ae3/src/convertor_tests.rs#L114) where `next_day_starts_at` is subtracted. In this PR's code it's changed to addition, so the subtraction must be done client-side (in Javascript) as explained in the comments.

The rest is correct.